pytube/request.py

```diff
@@ -17,7 +17,7 @@ def _execute_request(
     method: Optional[str] = None,
     headers: Optional[Dict[str, str]] = None,
 ) -> HTTPResponse:
-    base_headers = {"User-Agent": "Mozilla/5.0"}
```

Comment:
qq: what is rationale to add  `en-US,en` here?

Reply:
I am not from a country where english is spoken, so if I request for example a private video, the response would contain "Dieses Video ist Privat" instead of "This video is private". This fact breaks many features for non english countries like for example the last change in playlists or the upload date, because the regex-es used are made for americans. This header ensures the answer is sent in english.

Reply:
@tooxo very good point - thank you for the context 😄